jupyter_fsspec/handlers.py

```diff
@@ -251,6 +251,77 @@ def post(self):
         self.finish()
 
 
+# ====================================================================================
+# Handle Syncing Local and Remote filesystems
+# ====================================================================================
+class FilesystemSyncHandler(BaseFileSystemHandler):
```

Comment:
I would suggest that the FIlesystemManager should have the core functionality, and that the handler only take care of passing on the appropriate parameters and set the output status.